packages/pharos/src/components/popover/pharos-popover.scss

```diff
@@ -0,0 +1,44 @@
+@use '../../utils/scss/mixins';
+
+:host {
+  display: block;
+  position: fixed;
+  z-index: 140;
+  pointer-events: none;
+  top: 0;
+  left: 0;
+}
+
+.popover {
+  @include mixins.interactive-focus;
+
+  background-color: var(--pharos-color-ui-10);
+  position: relative;
+  margin: 0;
+  list-style-type: none;
+  padding-left: 0;
+  padding-inline-start: 0;
+  box-shadow: 0 1px 2px rgb(18 18 18 / 0.3), 0 4px 8px 3px rgb(18 18 18 / 0.15);
+  border-radius: var(--pharos-radius-base-standard);
+  visibility: hidden;
+  pointer-events: none;
+  opacity: 0;
+  transform: translate3d(0, 0, 0);
+  transition: visibility 0s var(--pharos-transition-duration-short),
+    opacity var(--pharos-transition-duration-short) linear;
+}
+
+:host([open]) .popover {
+  visibility: visible;
+  opacity: 1;
+  transition-delay: 0ms;
+  pointer-events: auto;
+}
+
+:host(:not([full-width])) .popover {
+  min-width: var(--pharos-dropdown-menu-size-width-min);
```

Comment:
This is a component token for the DropdownMenu and should not be used here.

Reply:
This hadn't been cleaned up from when I copied the dropdown-menu css. I was able to remove it without issue